### PR TITLE
add test case

### PR DESCRIPTION
**Pull request by LeungChiNan**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation and Context

add some case

## Description

add some case

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: diopi_test/python/conformance/diopi_functions.py [hunk 1]
```diff
@@ -2424,7 +2424,10 @@ def cumsum(input, dim, dtype=None):
     assert isinstance(dim, int), "dim should be int"
 
     sizeI = list(input.size().data)
-    assert dim < len(sizeI), "dim out of index"
```
Comment (yangbofun) [inline diopi_test/python/conformance/diopi_functions.py:2427]: 这里不用改动吧，一个tensor的shape的长度，不会为0。，例如a = torch.tensor([]); len(a.size())的值等于1.

File: diopi_test/python/conformance/diopi_functions.py [hunk 2]
```diff
@@ -2762,8 +2765,12 @@ def expand(input, size) -> Tensor:
 
 def unfold(input, dimension, size, step):
     sizeO = list(input.size().data)
-    sizeO[dimension] = int((sizeO[dimension] - size) / step + 1)
-    sizeO.append(size)
+
+    if len(sizeO) == 0:
```
Comment (yangbofun) [inline diopi_test/python/conformance/diopi_functions.py:2769]: 同上，这里应该改动的点是，input.size()函数。此时input.size()函数的返回值应该是[0], 其长度为1.

File: diopi_test/python/conformance/diopi_functions.py (hunk 1 above)
Comment (LeungChiNan) [reply diopi_test/python/conformance/diopi_functions.py:2427]: 这里是因为测例里考虑到了0-d的张量，所以加了个判断
image

File: diopi_test/python/conformance/diopi_functions.py (hunk 2 above)
Comment (LeungChiNan) [reply diopi_test/python/conformance/diopi_functions.py:2777]: 同上，考虑到了torch.randn(())的情况

File: diopi_test/python/conformance/conformance_test.py [hunk 3]
```diff
@@ -203,8 +203,8 @@ def test_uniform(input, start=0, end=1):
 
     def test_bernoulli(input, inplace=False, p=None):
         p_numpy = input.numpy()
-        p = p_numpy.mean() if p is None else p
         out = F.bernoulli(input, inplace, p)
+        p = p_numpy.mean() if p is None else p
```
[the remark this answers is not in the document]
Comment (LeungChiNan) [reply diopi_test/python/conformance/conformance_test.py]: input输入空tensor，mean会得出nan
现在改为先判断是否为空张量

**Review by NeosZhang — APPROVED** [review]

**Review by yangbofun — APPROVED** [review]

**Review by yangbofun — APPROVED** [review]

**Review by hellozmz — APPROVED** [review]

**Review by POI-WX — APPROVED** [review]

**Pull request merged**